src/components/Slider.tsx

```diff
@@ -85,11 +89,27 @@ const $Track = styled(Track)<{ $midpoint?: number }>`
       )
       0 0 / 0.6rem;
   }
+`;
 
-  ${({ $midpoint }) =>
+const $Tick = styled.span<{ $midpoint?: number; $light: boolean; $text?: string }>`
```

Comment:
nit: `$light` -> `$isLightMode`

Reply:
ah I see $light is just whether its the first tick in the tickArray. Can disregard